src/EnergyPlus/OutputProcessor.hh

```diff
@@ -74,6 +74,51 @@ struct EnergyPlusData;
 
 namespace OutputProcessor {
 
+    enum class eTimeStepType : int
+    {
+        Zone = 0,
+        HeatBalance = 1,
+        HVAC = 2,
+        System = 3,
+        Plant = 4
+    };
+    std::array<std::string_view, 5> constexpr sTimeStepType = {"Zone", "HeatBalance", "HVAC", "System", "Plant"};
```

Comment:
Added a constexpr array for getting a string representation of the enum.

Reply:
One pattern I think we should establish is the every enumeration ends in a value called `Num` which is the number of elements in the enumeration so that the string array can be defined as `std::array<std::string_view, eTimeStepType::Num>`.

Reply:
Will do that now.